datastore/additional_data/sources/org_types.py

```diff
@@ -0,0 +1,36 @@
+import re
+
+
+class OrgTypesSource(object):
+    """ This adds a well educated guess for the type of organisation the funding organisation is"""
+
+    ADDITIONAL_DATA_KEY = "fundingOrgType"
+    REGEX = 0
+    VALUE = 1
+
+    def __init__(self, *args, **kwargs):
+        self.org_types_rules = [
+            (re.compile("GB-(LAE|LAN|LAS|PLA)-.*", re.IGNORECASE), "Local Government"),
+            (re.compile("GB-GOR-.*", re.IGNORECASE), "Government"),
+            (
+                re.compile(
+                    "GB-COH-RC000766|360G-blf|GB-CHC-1036733|GB-GOR-PC390",
+                    re.IGNORECASE,
+                ),
+                "National Lottery",
+            ),
+        ]
```

Comment:
If I'm understanding this right then the later rules overwrite the earlier ones? If so I think we should add a comment that makes this explicit.